app/scripts/services/shardinfo.js

```diff
@@ -84,7 +84,7 @@ angular.module('shardinfo', [])
         SQLQuery.execute(recoveryStmt)
           .success(function (recoveryQuery) {
             var result = queryResultToObjects(recoveryQuery,
-                ['table_name', 'schema_name', 'recovery_stage', 'recovery_percent']);
+                ['table_name', 'schema_name', 'recovery_stage', 'recovery_percent', 'count']);
```

Comment:
afaik recovery_percent is not available anymore, you've replaced it with avg(recovery['size']['percent'])